instrumentation/src/main/java/io/opentelemetry/android/config/OtelRumConfig.java

```diff
@@ -119,4 +125,15 @@ public void disableNetworkChangeMonitoring() {
     public boolean isNetworkChangeMonitoringEnabled() {
         return this.networkChangeMonitoringEnabled;
     }
+
+    /** Adds a new initialization listener. It will be called during the RUM initialization. */
+    public void addInitializationListener(InitializationListener listener) {
+        if (!initializationListeners.contains(listener)) {
+            initializationListeners.add(listener);
+        }
+    }
+
+    public List<InitializationListener> getInitializationListeners() {
```

Comment:
A comment that is an unmodifiable list would be nice, maybe people will try to mutate directly from the list instead of calling `add`, etc.
Maybe adding a `remove` method as well? not sure if really needed.

Reply:
I like the idea of adding a comment to explain it. I think we could add a `remove` method in the future if needed, but so far I don't have a use case in mind for it.